internal/middleware/logging.go

```diff
@@ -14,11 +18,24 @@ func NewLoggingInterceptor(logger logger.Logger) grpc.UnaryServerInterceptor {
 	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
 		start := time.Now()
 
+		spanCtx := trace.SpanContextFromContext(ctx)
```

Comment:
Any reason why we shouldn't just use https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/instrumentation/google.golang.org/grpc/otelgrpc?

Reply:
Oh, good point. Thanks! Updated.

Reply:
I don't see the updates. Maybe you haven't pushed them yet?